Compiler.coffee

```diff
@@ -70,8 +70,12 @@ class Compiler
         props.className = className
 
       React.cloneElement child, props,
-        React.Children.map child.props.children, (child) =>
-          @traverseChild child, parentClass
+        if React.Children.count(child.props.children) is 1
```

Comment:
ok, yeah that makes sense, we don't necessarily want to remove the array wrap when it's there. I didn't want to do that because React advices using only their React.Children utilities for dealing with children. It's kind of an encapsulation so we don't rely on the data structure of props.children.